R/tm_a_regression.R

```diff
@@ -28,47 +28,90 @@
 #'
 #' @note For more examples, please see the vignette "Using regression plots" via
 #'   `vignette("using-regression-plots", package = "teal.modules.general")`.
-#' @export
 #'
 #' @examples
-#' # Regression graphs from selected response variable (BMRKR1) and
-#' # selected regressors (AGE)
+#' # general data example
+#' library(teal.widgets)
 #'
 #' data <- teal_data()
 #' data <- within(data, {
 #'   library(nestcolor)
-#'   ADSL <- teal.modules.general::rADSL
+#'   CO2 <- CO2
```

Comment:
@chlebowa if there is a way to remove `teal.modules.general::rADSL` complitely from examples, then we can maybe remove this whole dataset at all?

Reply:
remove this whole dataset at all from the package*?

Reply:
https://github.com/insightsengineering/teal.modules.general/issues/29#issuecomment-1952358103

Reply:
:)